### DISCUSSION TITLE
website https://app.coqui.ai/studio doesnt send verification email

### DISCUSSION THREAD

**Discussion by G-force78** [coqui-ai/TTS#2585] [General]
as described

**Comment by reuben** [comment]
Please email info@coqui.ai using the same email address you created your account with, so we can check it.

**Reply by G-force78** [reply]
> Please email [info@coqui.ai](mailto:info@coqui.ai) using the same email address you created your account with, so we can check it.

seemed to be a firefox issue chrome was fine
